src/Units.jl

```diff
@@ -231,9 +231,21 @@ for op in (:+, :-, :*, :/)
     @eval Base.$op(x::GeoUnit, y::Number) = $(op)(x.val, y)
     @eval Base.$op(x::Number, y::GeoUnit) = $(op)(x, y.val)
 
-    # Multiplying a GeoUnit with another one, returns a GeoUnit
+    # Multiplying/dividing/adding/subtracting a GeoUnit with another one, returns a GeoUnit
     @eval function Base.$op(x::GeoUnit{T1,U1}, y::GeoUnit{T2,U2}) where {T1,T2,U1,U2}
```

Comment:
Looks good to me. Perhaps we can remove `T1,U1,T2,U2` from the arguments type signatures since they are not used for anything in this function.